.travis.yml

```diff
@@ -1,33 +1,72 @@
 os: linux
 dist: focal
-language: c++
-env:
-  global:
-       - DEBIAN_FRONTEND: noninteractive
-addons:
-  apt:
-    packages:
-        - wget
-        - locales
-        - python3-pip
+language: minimal
+
+services:
+  - docker
 
-before_install:
-        - sudo locale-gen "en_US.UTF-8"
-        - export LANG="en_US.UTF-8"
-        - pip install git+https://github.com/antmicro/tuttest
-        - tuttest README.md unnamed0 | bash -
-jobs:
-  include:
-    - stage: SymbiFlow
-      install:
-            - tuttest README.md unnamed1 | bash -
+env:
+  # Testing QuickLogic toolchain on all OSes
+  - TOOLCHAIN=quicklogic      OS=ubuntu OS_VERSION=trusty
+  - TOOLCHAIN=quicklogic      OS=ubuntu OS_VERSION=xenial
+  - TOOLCHAIN=quicklogic      OS=ubuntu OS_VERSION=bionic
+  - TOOLCHAIN=quicklogic      OS=ubuntu OS_VERSION=eoan
+  - TOOLCHAIN=quicklogic      OS=ubuntu OS_VERSION=focal
+  - TOOLCHAIN=quicklogic      OS=centos OS_VERSION=6
+  - TOOLCHAIN=quicklogic      OS=centos OS_VERSION=7
+  - TOOLCHAIN=quicklogic      OS=centos OS_VERSION=8
+  # Testing Xilinx 7 Series toolchain on all OSes
+  - TOOLCHAIN=xilinx-series-7 OS=ubuntu OS_VERSION=trusty
+  - TOOLCHAIN=xilinx-series-7 OS=ubuntu OS_VERSION=xenial
+  - TOOLCHAIN=xilinx-series-7 OS=ubuntu OS_VERSION=bionic
+  - TOOLCHAIN=xilinx-series-7 OS=ubuntu OS_VERSION=eoan
+  - TOOLCHAIN=xilinx-series-7 OS=ubuntu OS_VERSION=focal
+  - TOOLCHAIN=xilinx-series-7 OS=centos OS_VERSION=6
+  - TOOLCHAIN=xilinx-series-7 OS=centos OS_VERSION=7
+  - TOOLCHAIN=xilinx-series-7 OS=centos OS_VERSION=8
 
-      script:
-            - tuttest README.md unnamed3 | bash -
-    - #
-      install:
-            - tuttest README.md unnamed2 | bash -
+install:
+ - if [ "$OS" = "ubuntu" ]; then export INSTALL_COMMAND="apt update && apt install -y"; fi
```

Comment:
If these packages need to be installed, then they should be part of the install instructions -- but I'm pretty sure a lot of these packages are unneeded?

Reply:
These packages are required to build openssl and python. See below why we are building python. I’ll open another PR adding the necessary info to the README.

Reply:
We shouldn't be building python here. This is the wrong solution to the problem. See my suggestion below.